src/server/game/Scripting/ScriptReloadMgr.cpp

```diff
@@ -868,8 +859,6 @@ class HotSwapScriptReloadMgr final
 
         Optional<fs::path> cache_path;
```

Comment:
Only the initialization of `cache_path = GenerateUniquePathForLibraryInCache(path)`